### ISSUE TITLE
Missing binding

### ISSUE THREAD

**Issue by myedibleenso** [clulab/reach#367]
I'm working through the missing events from the assembly corpus annotations and noticed that we no longer capture a binding in this sentence:

>Yang et al. also implicated a role for ERK in the regulation of FOXO3a by Mdm2 and provide evidence that FOXO3a phosphorylation by ERK through an unknown mechanism induces Mdm2 binding to FOXO3a     .

I'm not sure when this disappeared (I believe we were still finding it when https://github.com/clulab/reach/commit/60ae87a79ea079a5abdc2ba363085af726bff343 was pushed), but this seems like an obvious pattern and the graph doesn't look odd:

```
(28,induces,VBZ), (29,Mdm2,NN), (30,binding,NN), (31,to,TO), (32,FOXO3a,NN),

outgoing:
	27: (25,det) (26,amod)
	28: (19,mark) (21,nsubj) (30,dobj) (32,prep_to)
	29:
	30: (29,nn)
```

In the `controlled` of the activation below, we are now capturing "Mdm2 **binding**" as an `Entity`:

```
MENTION TEXT:  Mdm2 binding
LABELS:        List(Family, Equivalable, BioChemicalEntity, BioEntity, Entity, PossibleController)
DISPLAY LABEL: Family
	------------------------------
	RULE => ner-family-entities
	TYPE => CorefTextBoundMention
	------------------------------
	GROUNDING: KBResolution(mdm2binding, pfam, PF14918, )
	------------------------------

MENTION TEXT:  FOXO3a phosphorylation by ERK through an unknown mechanism induces Mdm2 binding
LABELS:        List(Positive_activation, ActivationEvent, ComplexEvent, Event, PossibleController)
DISPLAY LABEL: Positive_activation
	------------------------------
	RULE => Positive_activation_syntax_1_verb
	TYPE => CorefEventMention
	------------------------------

	------------------------------
	TRIGGER => induces
	controlled (List(Family, Equivalable, BioChemicalEntity, BioEntity, Entity, PossibleController)) => Mdm2 binding
	controller (List(Phosphorylation, AdditionEvent, SimpleEvent, Event, PossibleController)) => FOXO3a phosphorylation
	------------------------------
```

The crf is producing the following sequence labels for the last four tokens in this sentence:

```
B-Family  I-Family  O   B-Gene_or_gene_product  O
Mdm2      binding   to  FOXO3a                               .
```

@MihaiSurdeanu, any idea what may have changed?  I'm sure there are valid cases out there (ex. the "X binding protein"), but we weren't producing this label sequence before...

**Comment by myedibleenso** [comment]
Here is another example of missing bindings from PMC3351432:

>MP-1 interacts specifically with MEK1 and ERK1, enhances their interaction and recruits these complexes to late endosomes. MP-1 complex is anchored to the endosomal membrane by means of binding to the late-endosomal resident proteins, p14 and p18

**Comment by MihaiSurdeanu** [comment]
@myedibleenso: this was fixed in processors by the PR #105 "Improved validation of named entities in RuleNER". It was caused by a new entity "Mdm2-binding", which is tokenized into "Mdm2 binding", which caused the NER to over match. I modified the NER validator to detect if the "-ing" token is followed by "to", in which case the expansion to include "binding" is not accepted.

Also, I changed the NER validator to be dynamically applied during the search. Previously, it was applied just at the end, which is wrong: if the validation at the end failed, the whole NER matching fails, when, in reality, some other entity exists that starts at the same position (e.g., "Mdm2" is a valid protein).
